package.json

```diff
@@ -38,6 +38,7 @@
   "dependencies": {
     "axios": "^0.21.1",
     "form-data": "^2.3.3",
+    "ts-node": "^10.9.2",
```

Comment:
What is `ts-node` for?

Reply:
Not sure, I think it was added as dependency when I was doing audit fix